src/main/kotlin/org/wycliffeassociates/otter/common/domain/collections/GetCollections.kt

```diff
@@ -0,0 +1,17 @@
+package org.wycliffeassociates.otter.common.domain.collections
+
+import io.reactivex.Single
+import org.wycliffeassociates.otter.common.data.model.Collection
+import org.wycliffeassociates.otter.common.persistence.repositories.ICollectionRepository
+
+class GetCollections(private val collectionRepo: ICollectionRepository) {
```

Comment:
This feels like a useless abstraction over the collection repo?

Reply:
True. Better to just call the collection repo directly in the view models?